SPECS/systemd/systemd.spec

```diff
@@ -101,6 +104,10 @@ Just the definitions of rpm macros.
 Summary:        Development headers for systemd
 Requires:       %{name} = %{version}-%{release}
 Requires:       glib-devel
+
+Obsoletes:      systemd-bootstrap-devel <= %{version}-%{release}
```

Comment:
Should this be a 
```suggestion
Obsoletes:      systemd-bootstrap-devel < %{version}-%{release}
```

Reply:
No, but thanks for reminding me - we actually want to obsolete all versions and releases of `systemd-bootstrap`, since `systemd` is the one packages should be using, if present. I'll need to update this.